src/main/java/io/aiven/klaw/service/ServerConfigService.java

```diff
@@ -58,18 +59,21 @@ public void getAllProperties() {
     log.info("All server properties being loaded");
 
     List<ServerConfigProperties> listProps = new ArrayList<>();
-    List<String> allowedKeys =
-        Arrays.asList(
-            "spring.", "java.", "klaw.", "server.", "logging.", "management.", "endpoints.");
+    List<String> allowedKeys = Arrays.asList("spring.", "klaw.");
 
     if (env instanceof ConfigurableEnvironment) {
-      for (PropertySource propertySource : ((ConfigurableEnvironment) env).getPropertySources()) {
+      for (PropertySource<?> propertySource :
+          ((ConfigurableEnvironment) env).getPropertySources()) {
         if (propertySource instanceof EnumerablePropertySource) {
-          for (String key : ((EnumerablePropertySource) propertySource).getPropertyNames()) {
+          for (String key : ((EnumerablePropertySource<?>) propertySource).getPropertyNames()) {
 
             ServerConfigProperties props = new ServerConfigProperties();
             props.setKey(key);
-            if (key.contains("password") || key.contains("license")) {
+            if (key.contains("password")
```

Comment:
If I'm not mistaken String.contains is case sensitive. Depending on what is returned for "key", should that be considered here?

Reply:
Yes, but it may not be considered here. All the keys are already lower case now.

Reply:
Btw, thank you for the review.